content/docs/pipelines/standalone-deployment-gcp.md

```diff
@@ -22,7 +22,7 @@ export PIPELINE_VERSION={{% kfp-latest-version %}}
 kubectl apply -k github.com/kubeflow/pipelines//manifests/kustomize/env/dev?ref=$PIPELINE_VERSION
 ```
 
-2. Get the URL for the Kubeflow Pipelines UI :
+1. Get the URL for the Kubeflow Pipelines UI :
```

Comment:
The list formatting is broken in this section. To fix, indent the code snippets above and below this line.

Reply:
Thanks for the catch!